x-pack/plugins/security_solution/common/siem_migrations/model/api/rules/rule_migration.schema.yaml

```diff
@@ -175,3 +180,81 @@ paths:
                     description: Indicates the migration has been stopped.
         204:
           description: Indicates the migration id was not found running.
+
+  # Rule migration resources APIs
+
+  /internal/siem_migrations/rules/{migration_id}/resources:
+    post:
+      summary: Creates or updates rule migration resources for a migration
+      operationId: UpsertRuleMigrationResources
+      x-codegen-enabled: true
+      description: Creates or updates resources for an existing SIEM rules migration
+      tags:
+        - SIEM Rule Migrations
+        - Resources
+      parameters:
+        - name: migration_id
+          in: path
+          required: true
+          schema:
+            type: string
+            description: The migration id to attach the resources
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: array
+              items:
+                $ref: '../../rule_migration.schema.yaml#/components/schemas/RuleMigrationResourceData'
+      responses:
+        200:
+          description: Indicates migration have been created correctly.
```

Comment:
```suggestion
          description: Indicates migration has been created correctly.
```

Reply:
Good catch, actually it should be: 
`Indicates migration resources have been created or updated correctly`